views/partials/forms/option-group.html

```diff
@@ -15,11 +15,12 @@
                 required="true"
                 {{#toggle}} data-toggle="{{toggle}}"{{/toggle}}
                 {{#selected}} checked="checked"{{/selected}}
-                {{^error}}{{#hint}} aria-describedby="{{key}}-hint"{{/hint}}{{/error}}
+                {{^error}}{{#hint}} aria-describedby="{{key}}-{{value}}-hint"{{/hint}}{{/error}}
```

Comment:
This currently references a hint above. Are you saying this is a bug?

Reply:
I would think so - the data passed to the template in the 'hint' variable in this context comes from the option itself, not from the main field. If there is a hint present it should be pointing to the hint for the option (not previously present)